### ISSUE TITLE
React FE - Single player games always return isMainGameReady = true when polling getLobbyRefresh

### ISSUE THREAD

**Issue by LaustinSpayce** [Talishar/Talishar#261]
Before submitting a sideboard in a single player game the BE returns `isMainGameReady = true`, it should be `isMainGameReady = false` until the player has submitted their sideboard.

```
{"lastUpdate":"2","gameLog":"","playAudio":0,"theirHero":"DUMMY","theirHeroName":"Practice Dummy","theirName":"Player 2","theirNameColor":"","theirOverlayUrl":"","theirChannelLink":"","submitSideboard":"block","myPriority":true,"isMainGameReady":true,"canSubmitSideboard":true}
```

**Comment by LaustinSpayce** [comment]
In addition while queuing for a second player on my local machine:

`canSubmitSideboard: true`

While waiting for a second player to join

**Comment by OotTheMonk** [comment]
Fixed in 2b3c5b1a and 0e075f7a
